packages/blockchain-wallet-v4-frontend/src/data/actionTypes.ts

```diff
@@ -13,7 +13,7 @@ import * as middleware from './middleware/actionTypes'
 import * as modals from './modals/actionTypes'
 import * as modules from './modules/actionTypes'
 import * as preferences from './preferences/actionTypes'
-import * as prices from './prices/actionTypes'
+import { actions as prices } from './prices/slice'
```

Comment:
this line can be deleted since the actionTypes are inferred for us with redux-toolkit